go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/pashagolub/pgxmock v1.8.0
 	github.com/pashagolub/pgxmock/v2 v2.12.0
 	github.com/stretchr/testify v1.8.2
-	go.mongodb.org/mongo-driver v1.12.1
+	go.mongodb.org/mongo-driver v1.13.0
```

Comment:
problem was created because of https://github.com/mongodb/mongo-go-driver/pull/1347